### PR TITLE
updated the build image section, removed trademarks, minor edits

### PR DESCRIPTION
**Pull request by tanberry**
Made changes for 2.2 GA release, mostly updated the Building Agent section to add clarity

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -82,16 +84,20 @@ More information about managing your deployments is documented for [Kubernetes](
...
+Obtain the source code to build the agent from our [GitHub repository](https://github.com/logdna/logdna-agent-v2). You can either download the source files as a .zip file, or clone the repository.
+
+* Use the following commands to clone and then cd into this repository:
 
 ```
 git clone https://github.com/logdna/logdna-agent-v2.git
 cd logdna-agent-v2
 ```
 
+Next, select how you want to build the image: using Docker or an agent binary on Linux.
```
Comment (c-nixon) [inline docs/README.md:96]:
```suggestion
Next, select how you want to build the agent: as a docker image or as a Linux binary.
```

File: docs/README.md [hunk 2]
```diff
@@ -105,7 +111,9 @@ REPOSITORY              TAG                 IMAGE ID            CREATED
 logdna-agent-v2         dcd54a0             e471b3d8a409        22 seconds ago      135MB
 ```
 
-### Building agent binary on Linux
+You can also obtain the image and review our tagging scheme on [DockerHub](https://hub.docker.com/r/logdna/logdna-agent.)
```
Comment (c-nixon) [inline docs/README.md:114]:
```suggestion
You can also obtain the image and review our tagging scheme on [DockerHub](https://hub.docker.com/r/logdna/logdna-agent).
```

**Review by c-nixon — CHANGES_REQUESTED** [review]
Couple of tiny nits

File: docs/README.md [hunk 3]
```diff
@@ -20,21 +20,23 @@ The LogDNA agent is a fast, resource-efficient log collection client that forwar
   * [Running as Non-Root](#running-as-non-root)
```
Comment (alfeng6) [inline docs/README.md:20]: We should take this out for 2.2 GA Docs.

**Comment by alfeng6** [comment]
Can we also make sure to take out any reference to non-root in the`kubernetes.md` and `openshift.md` docs?

**Review by alfeng6 — CHANGES_REQUESTED** [review]
Main changes are changing our docs to not mention Non-root for our v2.2 GA docs.

File: docs/README.md (hunk 3 above)
Comment (c-nixon) [reply docs/README.md:20]: They'll disappear when I merge back to the 2.2 branch

**Review by c-nixon — APPROVED** [review]

**Review by tanberry — COMMENTED** [review]
ship it.....

**Pull request merged**